internal/pkg/manifest/lb_web_svc.go

```diff
@@ -232,6 +232,11 @@ func (s LoadBalancedWebService) applyEnv(envName string) (workloadManifest, erro
 
 // NetworkLoadBalancerConfiguration holds options for a network load balancer
 type NetworkLoadBalancerConfiguration struct {
+	PrimaryRoutingRule     NetworkLoadBalancerRoutingRule   `yaml:",inline"`
```

Comment:
oh yea this is awkward 😂 

It looks like `additional_rules` makes sense for ALB but for NLB this would be more like `additional_listeners` 

I think we might need to bite the bullet and renaming this field for NLB 😭 Perhaps, since this is a more advanced feature we can deem having two separate field names OK.


(imo we also need to rename the fields and types)